pkg/appmanager/validateResources.go

```diff
@@ -42,6 +44,19 @@ func (appMgr *Manager) checkValidConfigMap(
 		}
 		return false, nil
 	}
+	// This ensures that pool-only mode only logs the message below the first
+	// time we see a config.
+	rsName := formatConfigMapVSName(cm)
+	// Checking for annotation in VS, not iApp
+	if cfg.MetaData.ResourceType != "iapp" &&
+		cfg.Virtual.VirtualAddress != nil &&
+		cfg.Virtual.VirtualAddress.BindAddr == "" {
+		// Check for IP annotation provided by IPAM system
+		if _, ok := cm.ObjectMeta.Annotations[f5VsBindAddrAnnotation]; !ok {
+			log.Infof("No virtual IP was specified for the virtual server %s creating pool only.",
+				rsName)
```

Comment:
Don't forget to also verify that the config exists already, same as how you did it with Ingress.